presto-docs/src/main/sphinx/connector/inmemory.rst

```diff
@@ -0,0 +1,55 @@
+==================
+InMemory Connector
+==================
+
+The InMemory connector stores all data and metadata in RAM on workers
+and both are discarded when Presto restarts.
```

Comment:
Please add another warning that today there is no way to limit the memory this connector can use. There is also no way to check how much memory is already consumed by the connector on each node.